src/main/java/science/icebreaker/service/DeviceAvailabilityService.java

```diff
@@ -113,6 +113,7 @@ public void addDeviceAvailability(
         DeviceAvailability availabilityEntry = new DeviceAvailability();
         availabilityEntry.setAccount(account);
         availabilityEntry.setDevice(device);
+        availabilityEntry.setDisabled(false);
```

Comment:
The owner should be able to see all his entries. But maybe add this option as a method parameter?

Reply:
This is true. In fact, the owner must be able to see all his entries, otherwise he won't be able to enable them again. I'll fix that